test/net/sourceforge/kolmafia/request/PlaceRequestTest.java

```diff
@@ -74,7 +79,75 @@ void itShouldSetTheToolbeltAsAFreePullInTTT() {
     assertEquals(
         -1, freePulls.indexOf(toolbelt), "toolbelt should not be in Free Pulls after TTT fades");
   }
-}
 
-// Generated with love by TestMe :) Please report issues and submit feature requests at:
-// http://weirddev.com/forum#!/testme
+  @Nested
+  class Speakeasy {
+    @BeforeEach
+    public void initializeCharPrefs() {
+      KoLCharacter.reset("PlaceRequestTestFakeSotVisit");
+      KoLCharacter.reset(true);
+    }
+
+    @AfterEach
+    public void resetCharAndPrefs() {
+      KoLCharacter.reset("");
+      KoLCharacter.reset(true);
+      KoLCharacter.setUserId(0);
+    }
+
+    @Test
+    public void itShouldGetParcelLocationFromFirstVisit() {
+      String prefName = "_sotParcelLocation";
+      assertEquals("", Preferences.getString(prefName), "Preference already set.");
+      var req = new GenericRequest("place.php?whichplace=speakeasy&action=olivers_sot");
```

Comment:
As coded, why are you making a GenericRequest?
You set a responseText and then remove it.
You never run the request.

We have many examples of setting up a fake http server and actually running it, but if you simply want to call a responsetext parser which is not an instance method, you are not using tge request.